torchgeo/trainers/chesapeake.py

```diff
@@ -45,31 +58,46 @@ class ChesapeakeCVPRSegmentationTask(LightningModule):
 
     def config_task(self) -> None:
         """Configures the task based on kwargs parameters passed to the constructor."""
+        if self.hparams["class_set"] == 5:
+            num_classes = 5
+            classes = [1, 2, 3, 4]
+        elif self.hparams["class_set"] == 7:
+            num_classes = 7
+            classes = [1, 2, 3, 4, 5, 6]
+        else:
+            raise ValueError("'class_set' must be either 5 or 7")
```

Comment:
This could be replaced with:
```python
num_classes = self.hparams["class_set"]
assert num_classes in [5, 7]
classes = list(range(num_classes))
```